### DISCUSSION TITLE
Why adding and removing bookmark is so complicated?

### DISCUSSION THREAD

**Discussion by patelrohan** [alefragnani/vscode-bookmarks#577] [Q&A] [answered]
I am wondering if this action can be more intuitive. I just installed the extension by seeing highest voted answer on Stackoverflow and I just can not figure out how this works. I did create a bookmark with label/description but nothing shows up in sidebar.

Also, I am having hard time following the gifs in how to Get Started Section. Have you considered making a small, simple video instead? It might be helpful for people with cognitive disability. 

![Screenshot 2022-12-20 122443](https://user-images.githubusercontent.com/1224661/208728446-938cae02-8aba-497e-8cd5-7df5eb925486.jpg)

Update: 

- I switched to tree view and now I see something but not really what I expected. It just shows file name with long path next to it. But, no bookmark(s) I have created and labeled. 

![bookmark](https://user-images.githubusercontent.com/1224661/208731481-4c649171-ffa7-4174-8184-a556df83c229.jpg)

- Below is a simple interface I was expecting to see for bookmark management. 

![book skeytch](https://user-images.githubusercontent.com/1224661/208731387-824754e7-0840-46f2-a057-571fd9b703e0.jpg)

**Comment by alefragnani** [comment, marked as answer]
Hi @patelrohan ,

Sorry if you feel uncomfortable with the extension, but there is nothing _complicated_ in its base use. In fact, right the opposite, as it follows most of the VS Code UX Guidelines.

Every command available to you is accessible via `Command Palette`, and the main commands (as `Toggle`) has a shortcut defined. The Side Bar is there to present you all bookmarks in the workspace, and provides you a few options to customize how they are organized/displayed.

* About your prints, the _missing bookmark in the Side Bar_, you may be facing the error #397 . 
* About the _long file path only with no bookmarks in it_, the bookmarks are there, but the file is _collapsed_. You have to _expand_ the file, so its bookmarks will be displayed similar to the Post It picture you attached.

The commands are here https://github.com/alefragnani/vscode-bookmarks#available-commands, and sample Side Bar is here https://github.com/alefragnani/vscode-bookmarks#side-bar.

I’m preparing a `Walkthrough` for upcoming releases, so it would be even easier for new users to use the extension. I’ll post here once it is available, so you could take a look

Hope this helps

**Reply by patelrohan** [reply]
Thank you. Maybe I am confused because I am encountering bug you mentioned. I explanded the file but nothing in sidebar. Still empty.

![image](https://user-images.githubusercontent.com/1224661/209017878-ddd72a97-5127-4cbb-828a-94e6155e6687.png)
